Failed to Compile

Hi, 
I can´t compile this mod. 

```
/srv/azerothcore-wotlk/modules/mod-npc-enchanter/src/npc_enchanter.cpp:1116:35: fatal error: no member named 'SendNotification' in 'WorldSession'
10:00:17               player->GetSession()->SendNotification("Something went wrong in the code. It has been logged for developers and will be looked into, sorry for the inconvenience.");
10:00:17               ~~~~~~~~~~~~~~~~~~~~  ^
10:00:17   1 error generated.
10:00:17   gmake[2]: *** [modules/CMakeFiles/modules.dir/build.make:132: modules/CMakeFiles/modules.dir/mod-npc-enchanter/src/npc_enchanter.cpp.o] Error 1
10:00:17   gmake[2]: *** Waiting for unfinished jobs....
10:00:18   gmake[1]: *** [CMakeFiles/Makefile2:1100: modules/CMakeFiles/modules.dir/all] Error 2
10:00:18   gmake[1]: *** Waiting for unfinished jobs....
```

I think it´s caused by this commit in core https://github.com/azerothcore/azerothcore-wotlk/commit/e4df159f5c370e1a1890a6a999b33b30d72eb55e

--- a/src/npc_enchanter.cpp
+++ b/src/npc_enchanter.cpp
@@ -1113,7 +1113,7 @@ public:
 
         if (!enchantid)
         {
-            player->GetSession()->SendNotification("Something went wrong in the code. It has been logged for developers and will be looked into, sorry for the inconvenience.");
+        	ChatHandler(player->GetSession()).SendNotification("Something went wrong in the code. It has been logged for developers and will be looked into, sorry for the inconvenience.");
             player->PlayerTalkClass->SendCloseGossip();
             creature->HandleEmoteCommand(EMOTE_ONESHOT_LAUGH);
             return;
@@ -1128,17 +1128,17 @@ public:
         // Random enchantment notification
         if (roll > 0 && roll < 33)
         {
-            player->GetSession()->SendNotification("|cff00ff00Beauregard's bony finger crackles with energy when he touches |cffDA70D6%s|cff00ff00!", item->GetTemplate()->Name1.c_str());
+            ChatHandler(player->GetSession()).SendNotification("|cff00ff00Beauregard's bony finger crackles with energy when he touches |cffDA70D6%s|cff00ff00!", item->GetTemplate()->Name1.c_str());
             // creature->HandleEmoteCommand(EMOTE_ONESHOT_POINT);
         }
         else if (roll > 33 && roll < 75)
         {
-            player->GetSession()->SendNotification("|cff00ff00Beauregard holds |cffDA70D6%s |cff00ff00up in the air and utters a strange incantation!", item->GetTemplate()->Name1.c_str());
+            ChatHandler(player->GetSession()).SendNotification("|cff00ff00Beauregard holds |cffDA70D6%s |cff00ff00up in the air and utters a strange incantation!", item->GetTemplate()->Name1.c_str());
             // creature->HandleEmoteCommand(EMOTE_ONESHOT_CHEER);
         }
         else
         {
-            player->GetSession()->SendNotification("|cff00ff00Beauregard concentrates deeply while waving his wand over |cffDA70D6%s|cff00ff00!", item->GetTemplate()->Name1.c_str());
+            ChatHandler(player->GetSession()).SendNotification("|cff00ff00Beauregard concentrates deeply while waving his wand over |cffDA70D6%s|cff00ff00!", item->GetTemplate()->Name1.c_str());
             // creature->HandleEmoteCommand(EMOTE_ONESHOT_WAVE);
         }
         creature->CastSpell(player, 12512); // enchantment visual

